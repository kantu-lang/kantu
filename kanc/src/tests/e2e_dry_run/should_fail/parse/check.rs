use super::*;

#[test]
fn question_checkee() {
    let path = concat_paths(
        file!(),
        checked_path!(
            "../../../sample_code/should_fail/single_file/parse/check/question_checkee.k"
        ),
    );
    let output =
        get_manifest_path_and_backslash_normalized_output(vec![DUMMY_EXEC_PATH, "--file", &path]);
    insta::assert_debug_snapshot!(output);
}
