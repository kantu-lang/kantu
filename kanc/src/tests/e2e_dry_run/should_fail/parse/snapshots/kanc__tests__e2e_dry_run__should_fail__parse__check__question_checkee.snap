---
source: src/tests/e2e_dry_run/should_fail/parse/check.rs
expression: output
---
"Error: [E0305] Could not parse file: Unexpected token `?` on <CARGO_MANIFEST_DIR>/src/tests/sample_code/should_fail/single_file/parse/check/question_checkee.k:5:17."
